common-pg/src/main/scala/com/socrata/pg/store/PGSecondaryDatasetMapWriter.scala

```diff
@@ -25,6 +26,7 @@ class PGSecondaryDatasetMapWriter[CT](override val conn: Connection,
       |VALUES (?, ?)
     """.stripMargin
   private val deleteQueryInternalNameMap = "DELETE FROM dataset_internal_name_map where dataset_system_id = ?"
+  private val enableQueryInternalNameMap = "UPDATE dataset_internal_name_map SET disabled = ? WHERE dataset_system_id = ?"
 
```

Comment:
Disabled was newly introduced last sprint.  It allows us to reject soql reads.  Write should be able to continue unaffected.